quantus_sdk/lib/src/services/migration_service.dart

```diff
@@ -59,9 +60,12 @@ class MigrationService {
         accountId: data.newAccountId,
       );
 
```

Comment:
See here - addAccount can crash and I suspect this is what happened. 

saveAccounts does not crash.